.github/workflows/qa-integrate.yml

```diff
@@ -285,7 +351,7 @@ jobs:
           path: qa-tester
 
       - name: Setup Google Cloud SDK
-        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        uses: google-github-actions/setup-gcloud@master
```

Comment:
Oh, why the switch here?

Reply:
The previous action was deprecated, see https://github.com/google-github-actions/setup-gcloud/blob/b94035a6aac71cdacf6e682bbbef2e927960f8ac/setup-gcloud/README.md.